documentation/docusaurus_plugins/docusaurusWebpack5Plugin.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Adapted from https://gist.github.com/sibelius/24f63eef7f43b15dc73c4a0be11bbef8
+ */
+
+const webpack = require('webpack');
+
+module.exports = function (context, options) {
+  return {
+    name: 'docusaurus-webpack-plugin',
+    // eslint-disable-next-line
+    configureWebpack(config, isServer, utils) {
```

Comment:
Is it possible to add this into webpack.config similar to viewer/examples to align, your thoughts!

Reply:
I don't think so, because Docusaurus has its own internal webpack config, which we are only able to change using plugins like this.